js/sites.js

```diff
@@ -303,10 +302,10 @@
 
     _ignoreHoverEvents(element) {
       element.addEventListener("mouseover", () => {
-        this.cell.node.setAttribute("ignorehover", "true");
+        this.cell.node.data.ignorehover = true;
```

Comment:
Nit: It should probably be called "ignoreHover" in JS and reflected as "data-ignore-hover" in CSS. That also matches the surrounding method.